File: pgm/ABC_Java/ABC_335_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final char[] S = reader.readLine().toCharArray();
        final char four = '4';
        S[S.length - 1] = four;
        System.out.println (String.valueOf (S));
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_335_A.kt
// import java.time.*
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.util.StringTokenizer

// import kotlin.math.*

const val sThree: String = "3"
const val sFour: String = "4"
const val blank: String = ""

fun main() {
    val S: String = readln()
    println(arrayOf(S.removeSuffix(sThree), sFour).joinToString(separator = blank))
    kotlin.system.exitProcess(0)
}